.gitignore

```diff
@@ -31,9 +28,8 @@ ObjectStore
 .gradle
 docker/distroless/bazel-*
 /.apt_generated_tests/
-quarkus.log
-quarkus.log*
-replay_*.logß
+*.log
+*.log*
```

Comment:
@gsmet Are you okay with this? Seems fine to me, but since it impacts more than my little part of the project...

Reply:
Yeah, I had seen it and I think we can try this approach and see if it becomes annoying.

Reply:
Taken offline, Guillaume is okay to try. Let's!